hood/crypto_trading/parse.py

```
@@ -0,0 +1,113 @@
+    TYPE_CHECKING,
+    TypeVar,
+    cast,
+)
+
+import requests
+
+from .. import schema as _schema
+
+
+if TYPE_CHECKING:
+    from _typeshed import DataclassInstance  # noqa
```

Ruff (0.14.10)

Unused blanket `noqa` directive

Remove unused `noqa` directive

(RUF100)